core/include/usFrameworkEvent.h

```diff
@@ -95,31 +99,28 @@ class US_Core_EXPORT FrameworkEvent
     FRAMEWORK_INFO = 0x00000020,
 
     /**
-     * The Framework has stopped.
-     *
+     * The Framework has been stopped.
      * <p>
-     * This event is fired when the Framework has been stopped because of a stop
-     * operation on the system bundle. The source of this event is the System
-     * Bundle.
+     * The Framework's
+     * \link BundleActivator::Stop(BundleContext*) BundleActivator Stop\endlink method
+     * has been executed.
      */
```

Comment:
This looks like a merge error. The original documentation is according to the specs and should be retained.